e2e/docker-compose.yml

```diff
@@ -20,33 +20,23 @@ services:
     environment:
       KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
-      KAFKA_LISTENERS: 'PLAINTEXT://kafka:9092'
-      KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT://kafka:9092'
+      KAFKA_LISTENERS: 'PLAINTEXT://0.0.0.0:9092,PLAINTEXT_INTERNAL://kafka:29092'
+      KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT://localhost:9092,PLAINTEXT_INTERNAL://kafka:29092' # Advertise both listeners
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: 'PLAINTEXT:PLAINTEXT,PLAINTEXT_INTERNAL:PLAINTEXT'
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
       KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
       KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
       KAFKA_DEFAULT_REPLICATION_FACTOR: 1
       KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
       KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'true'
+      KAFKA_CREATE_TOPICS: 'erc20_exporter_test:1:1'
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     logging:
       driver: none
-  exporter:
-    build:
-      context: ../
-      dockerfile: docker/Dockerfile
-      args:
-        NODE_ENV: development
+  kafka-init:
+    image: wurstmeister/kafka
```

Comment:
For this test we need one kakfa topic to exist. We used to do it inside the test but it complicates the logic there. Now do it with an extra container here.